src/App.js

```diff
@@ -29,6 +31,16 @@ class App extends React.Component {
         <Experiment settings={{
           timeline: tl,
           on_data_update: (data) => {
+            if (firebase) {
+              if (data.trial_index === 1 ) {
```

Comment:
isn't there an issue sometimes where we don't have the patient ID on the first trial? and don't get it until a few screens in?

Reply:
first trial is 0. 1 has the patient id. I guess it depends on the task.